scripts/module/admin.nut

```diff
@@ -84,11 +84,9 @@ function onPlayerCommand( player, cmd, text ) {
       local plr = FindPlayer( id );
       if ( plr ) {
         local reason = GetTok( text, " ", 2 NumTok( text, " " ) );
-        MessagePlayer( "wth...." + reason, player );
-        if ( reason ) {
-          Message( "[#FF0000]Admin " + player.Name + " kicked " + plr.Name + " for reason: " + reason );
-          KickPlayer( plr );
-        }
+	Message( "[#FF0000]Admin " + player.Name + " kicked " + plr.Name +
+		 " for reason: \"" + reason + "\"");
```

Comment:
this will probably look like garbage if reason is 0.

I suggest making the second part optional (by appending, or not appending the reason).